README.md

```diff
@@ -90,13 +120,15 @@ var app = express();
 Describe the differences between Express and Rails as backend frameworks.
 
 ```text
+Rails is a Ruby-based framework that automatically creates an explicit file structure and comes with quite a bit of built-in data-handling conventions "inside the box". Express is a JavaScript-based framework that is much more flexible in terms of file structure and naming conventions but the flipside of that flexibility is that it requires more input and decision making on the part of the developer.
 
+As an aside, I'm wondering how that structure vs flexibility tradeoff plays out when sharing code. I would think a dev coming to someone else's Rails app would have an easier time knowing where to find everything than if it was an Express app. True?
 ```
```

Comment:
👍  As someone who hasn't necessarily worked on a "dev" team, I can still say yes, it feels a little easier to find your way around a Rails app. I'm thinking specifically about working with the instructors on Garnet, and at least understanding where things are